### PR TITLE
Replaces js with translation files

### PR DESCRIPTION
**Pull request by liviacarolgouvea**

### CODE REVIEW

File: CspThemePlugin.inc.php [hunk 1]
```diff
@@ -12,7 +12,7 @@ public function init() {
         $this->setParent('bootstrapthreethemeplugin');
         //$this->modifyStyle('stylesheet', array('addLess' => array('styles/index.less')));
         $this->addStyle('child-stylesheet', 'styles/index.less');
-				$this->addScript('csp', 'js/index.js');
+		//$this->addScript('csp', 'js/index.js');
```
Comment (vitormattos) [inline CspThemePlugin.inc.php:15]: is necessary maintain this comment?

File: CspThemePlugin.inc.php [hunk 2]
```diff
@@ -12,7 +12,7 @@ public function init() {
         $this->setParent('bootstrapthreethemeplugin');
         //$this->modifyStyle('stylesheet', array('addLess' => array('styles/index.less')));
```
Comment (vitormattos) [inline CspThemePlugin.inc.php:13]: Unchanged line, but, is necessary maintain this line commented?

**Comment by vinicios-gomes** [comment]
Is the translation file returning the acronym for each language?
Requested to display the acronym and not the language full name.
Js file is just for that.

File: CspThemePlugin.inc.php (hunk 2 above)
Comment (liviacarolgouvea) [reply CspThemePlugin.inc.php:13]: I kept it in case it was necessary to use the js file for new tasks

**Pull request merged**